Source/Scene/Expression.js

```diff
@@ -144,6 +158,18 @@ define([
                 node.evaluate = node._evaluateEquals;
             } else if (node._value === '!==') {
                 node.evaluate = node._evaluateNotEquals;
+            } else if (node._value === '<') {
```

Comment:
1. Yes, we should throw them at runtime.  When do we need to wait until runtime?  Just for variables?  I suspect we will know everything else at compile time.

2. Those can be syntax errors.  GLSL has `any` and `all` functions, e.g., `any(bvec3(true, false, false)) == true`, but we don't need to introduce all that until we actually need it.  By the way, it is worth a few minutes to check out GLSL, see Pages 3 and 4: https://www.khronos.org/files/webgl/webgl-reference-card-1_0.pdf